### PR TITLE
allow backtest ploting

### PR DESCRIPTION
**Pull request by xmatthias**
## Summary
Allow plotting of backtest data (other than the profit graph, which i don't find very comfortable for Strategy development).

The script already exposed the --export-filename parameter - but did not use it but loaded trades only from the database eventually.

## what's new

- load files from exported backtest-trades
- Add option to modify the plot-limit - should someone want to plat a full month of 15m data (16Meg in my case, but it works).
- extend format of backtest to support plotting (some fields were missing). This should also allow better anlysis as open/close rate utilized be compared to real values (when comparing to dry-run/live results).

### CODE REVIEW

File: freqtrade/optimize/backtesting.py [hunk 1]
```diff
@@ -115,11 +117,13 @@ def _generate_text_table(self, data: Dict[str, Dict], results: DataFrame) -> str
 
     def _store_backtest_result(self, recordfilename: Optional[str], results: DataFrame) -> None:
 
-        records = [(trade_entry.pair, trade_entry.profit_percent,
-                    trade_entry.open_time.timestamp(),
-                    trade_entry.close_time.timestamp(),
-                    trade_entry.open_index - 1, trade_entry.trade_duration)
-                   for index, trade_entry in results.iterrows()]
+        # columns = ["pair", "profit", "opents", "closets", "index", "duration",
```
Comment (gcarq) [inline freqtrade/optimize/backtesting.py]: Seems like a debugging leftover, if not please write a proper comment

Comment (xmatthias) [reply freqtrade/optimize/backtesting.py]: Love a good Review!

moved this to the documentation (including a code-snipped for "manual" analysis"...)

**Comment by xmatthias** [comment]
@gcarq had a chance to check / verify the plot?

**Comment by gcarq** [comment]
LGTM

**Pull request merged**